orchagent/neighorch.cpp

```diff
@@ -267,6 +269,13 @@ bool NeighOrch::addNextHop(const NextHopKey &nh)
     next_hop_attrs.push_back(next_hop_attr);
 
     sai_object_id_t next_hop_id;
+
+    if (ctx.bulk_op)
```

Comment:
Just to clarify, the bulk operation is done only from mux context, right? or is it becoming the default flow with this change?

Reply:
Yes, currently the bulk_op flag is only set while processing mux neighbors during switchover